templates/new_request_page.hbs

```diff
@@ -19,6 +19,13 @@
   </h1>
 
   <div id="main-content" class="form">
-    {{request_form wysiwyg=true}}
+    <form data-form action="{{request_form_data.action}}" method="{{request_form_data.http_method}}" accept-charset="{{request_form_data.accept_charset}}">
```

Comment:
It might make it cleaner if we change the scope to the object itself using `with`:
```hbs
{{#with request_form}}
  <form data-form action="{{action}}" method="{{http_method}}" accept-charset="{{accept_charset}}">
    {{#if errors}}
      <p>{{errors}}</p>
    {{\if}}
  </form>
{{/with}}
```